Config.cpp

```diff
@@ -84,25 +99,25 @@ namespace {
             ssConfig.Fullscreen = (result == "NO");
         }
         if (line.find("[SEGMENTSIZE_XY") != string::npos) {
-            int value = parseIntFromLine("SEGMENTSIZE_XY", line);
-            if (value < 1) {
-                value = DEFAULT_SIZE;
-            }
-            if (value > 100) {
-                value = 100;
-            }
-            //plus 2 to allow edge readings
-            ssState.Size.x = value + 2;
-            ssState.Size.y = value + 2;
+            bool autosize = (parseStrFromLine("SEGMENTSIZE_XY", line) == "AUTO");
```

Comment:
why isn't this inside of `getSegmentSizeConfig`? it looks like duplicated boilerplate across all three usages